init.c

```diff
@@ -73,6 +73,9 @@ bool write_bw_log = false;
 bool read_only = false;
 int status_interval = 0;
 
+int ss_check_interval = STEADYSTATE_MSEC;
+
```

Comment:
One extra empty line here

Reply:
I would move this declaration next to where `steadystate_enabled` is declared in steadystate.c in order to have all of this in the same place.